tests/unittest/_torch/auto_deploy/unit/multigpu/test_ad_allreduce_strategies.py

```
@@ -54,6 +57,49 @@ def timeout_handler(signum, frame):
+
+        if torch.cuda.is_available():
+            # Create dummy tensors to trigger kernel JIT compilation
+            with torch.no_grad():
+                device = torch.device("cuda:0")
+
+                # Trigger page kernel compilation
+                try:
+                    # Force module loading (this triggers JIT compilation)
+                    _ = flashinfer.page.gen_page_module()
+                except Exception:
+                    pass  # Ignore errors - the import itself should trigger JIT
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)